.github/actions/spelling/allow.txt

```diff
@@ -0,0 +1,169 @@
+ACLs
+ACR
+AMD
+AWS
+Alpstein
+Balfrin
+Broyden
+CFLAGS
+CHARMM
+CHF
+COSMA
+CPE
+CPMD
+CSCS
+CWP
+CXI
+capstor
+Ceph
+Containerfile
+DNS
+EDF
+EDFs
+EDFs
+EMPA
+ETHZ
+Ehrenfest
+Errigal
+FFT
+firecrest
+Fock
+GAPW
+GCC
+GGA
+GPFS
+GPG
+GPU
+GPUs
+GPW
+GROMACS
+GTL
+Gaussian
+Google
+HDD
+HPC
+HPCP
+HPE
+HSN
+Hartree
+iopsstor
+Jax
+Jira
+Keycloak
+LAMMPS
+LDA
+LOCALID
+LUMI
+Libc
+Linaro
+Linux
+MFA
+MLP
+MNDO
+MPICH
+MPS
+MeteoSwiss
+NAMD
+NICs
+NVIDIA
+NVMe
+OTP
+OTPs
+PASC
+PBE
+PDUs
+PID
+PMPI
+POSIX
+Parrinello
+Piz
+Plesset
+Pulay
+RCCL
+RDMA
+RESTful
+ROCm
+RPA
+Roboto
+Roothaan
+SSHService
+STMV
+Scopi
+TOTP
+UANs
+UserLab
+VASP
+Waldur
+Wannier
+XDG
+aarch
+aarch64
+acl
+biomolecular
+bristen
+bytecode
+clariden
+concretise
+concretizer
```

Comment:
Note that I haven't made any judgment about e.g. `z` vs `s` in words like these. Feel free to bikeshed as much as you want about what goes into this file, but let's keep it practical.